rbac/management/permissions/workspace_access.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Defines the Audit Log Access Permissions class."""
+from rest_framework import permissions
+
+
+class WorkspaceAccessPermission(permissions.BasePermission):
+    """Determines if a user is an Account Admin."""
+
+    def has_permission(self, request, view):
+        """Check permission based on Account Admin property."""
+        return request.user.admin
```

Comment:
See comment below – before production I think we'll want this to use the Relations API itself to check this.

Reply:
https://github.com/RedHatInsights/insights-rbac/pull/1134#discussion_r1662075500